contact.Rmd

```diff
@@ -6,4 +6,10 @@ output:
     toc_float: false
 ---
 
-To get in touch, send us an email to [mail@covid19scenariohub.eu](mailto:mail@covid19scenariohub.eu).
+For general information about the European Scenario Hub, our [Github repository](https://github.com/covid19-forecast-hub-europe/covid19-scenario-hub-europe/issues/new) is typically the most up to date source. For updates on new scenario rounds, upcoming meetings, and results, all are welcome to join the European Scenario Hub [mailing list](https://groups.google.com/u/0/g/euro-covid19-scenario-hub). Please include your name and a brief sentence about your interest in the Hub (so that we can exclude bots). 
```

Comment:
Should we mention slack as well?

Reply:
Yes - would you suggest adding the invite link directly?

Reply:
Probably not but we should mention it exists. I think there is also a link where they can request access, like a google group. But maybe I've dreamt it :shrug:

Reply:
It looks like Slack links expire after 30 days - at least from what I can see in Slack. So i've just added a line mentioning it. Thanks for flagging!